book/models.py

```diff
@@ -15,4 +15,11 @@ class Books(models.Model):
     author = models.CharField(max_length=128)
     published_year = models.CharField(max_length=4)
     status = models.CharField(max_length=9, choices=STATUS, default=AVAILABLE)
-    ordered_by = models.ForeignKey(CustomUser, on_delete=models.SET_NULL, null=True, blank=True)
\ No newline at end of file
+    ordered_by = models.ForeignKey(CustomUser, on_delete=models.SET_NULL, null=True, blank=True)
+
+
+class BookCart(models.Model):
```

Comment:
This model is wrong,
In a cart we just need a book and user foreign key, And the status of the cart